tripal_chado/src/Plugin/ChadoBuddy/ChadoCvtermBuddy.php

```diff
@@ -160,8 +170,9 @@ public function getCv(array $conditions, array $options = []) {
    *     - db.urlprefix
    *     - db.url
    *     - buddy_record = a ChadoBuddyRecord can be used
-   *       in place of or in addition to other keys
-   * @param array $options (Optional)
+   *       in place of or in addition to other keys.
+   * @param array $options
+   *   (Optional)
    *   Associative array of options.
    *     - 'case_insensitive' - a single key, or an array of keys
    *                            to query case insensitively.
```

Comment:
Options should not be indented like this and should follow the format 
`- OPTION: definition.`

```suggestion
   * @param array $options
   *   (Optional) Associative array of options.
   *     - case_insensitive: a single key, or an array of keys to allow 
   *       query conditions to match ignoring the case of the value.
```